app/src/renderer/js/cropper.js

```diff
@@ -37,6 +39,10 @@ document.addEventListener('DOMContentLoaded', () => {
     if (event.keyCode === ESC_KEY_CODE) {
       autoDestroy();
     }
+
+    if (event.keyCode === ENTER_KEY_CODE || SPACE_KEY_CODE) {
```

Comment:
Use https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key instead, and you don't need to have constants for the different keys ;)